renku/core/session/renkulab.py

```diff
@@ -422,21 +434,27 @@ def session_stop(self, project_name: str, session_name: Optional[str], stop_all:
 
         return all([response.status_code == 204 for response in responses]) if responses else False
 
-    def session_open(self, project_name: str, session_name: str, ssh: bool = False, **kwargs) -> bool:
+    def session_open(self, project_name: str, session_name: Optional[str], ssh: bool = False, **kwargs) -> bool:
         """Open a given interactive session.
 
         Args:
             project_name(str): Renku project name.
-            session_name(str): The unique id of the interactive session.
+            session_name(Optional[str]): The unique id of the interactive session.
             ssh(bool): Whether to open an SSH connection or a normal browser interface.
         """
-        sessions = self.session_list("", None)
+        sessions = self.session_list(project_name="")
         system_config = SystemSSHConfig()
         name = self._project_name_from_full_project_name(project_name)
         ssh_prefix = f"{system_config.renku_host}-{name}-"
 
+        if not session_name:
+            if len(sessions) == 1:
+                session_name = sessions[0].id
+            else:
+                return False
```

Comment:
Same here with a warning

Reply:
We raise an error in `core/session/session.py` in this case: `raise errors.ParameterError("Session name is missing")`